Nemt/nemt.cpp

```diff
@@ -8,19 +8,27 @@ Nemt::Nemt(int cl) {
     counter = 0;
 }
 
+inline int Nemt::clear_return(int return_value) {
```

Comment:
No idea, it should probably explain what you want to do using this function. Also please note that it should be defined as outside of class or declared in class as well.